mozetl/taar/taar_utils.py

```diff
@@ -19,6 +19,7 @@
 AMO_DUMP_KEY = 'telemetry-ml/addon_recommender/addons_database.json'
 
 AMO_WHITELIST_KEY = 'telemetry-ml/addon_recommender/whitelist_addons_database.json'
+AMO_CURATED_WHITELIST_KEY = 'telemetry-ml/addon_recommender/only_guids_top_200.json'
```

Comment:
As far as I can tell, test_taar_locale was reading from the wrong (fake) s3 bucket, as of 0ddf9a19eeb77574c558e57589daf0066e7252ce. I'll try to make it more obvious what's really happening.

Reply:
Ok, I am clearly not the right person to solve this.

Answering questions:
(1) No race condition, just about writing the file to the correct place so it can be read later
(2) No, it's implicitly read by the `taar_locale.generate_dictionary` later in the file. The unit test is failing because the file we were expecting to be able to read is no longer in the correct place.
(3) These aren't really utility functions, but functions to read/write some kind of intermediary amo-type data structures. There is a bunch of related code and defines in `mozetl/taar/taar_amowhitelist.py` which duplicates some of what's written here. Likely some refactoring is in order.

I would like to just land this patch as it gets the tests working again. Presumably someone more familiar with this code like @crankycoder can clean it up when time allows.

Reply:
Go it. I missed 2) because I was grepping for the variable name.